blockchain/chain.go

```diff
@@ -1647,29 +1651,23 @@ func (b *BlockChain) connectBestChain(node *blockNode, block, parent *dcrutil.Bl
 			var err error
 			fork, err = b.index.PrevNodeFromNode(fork)
 			if err != nil {
-				return false, err
+				return 0, err
 			}
 		}
 
 		// Log information about how the block is forking the chain.
 		if fork.hash == node.parent.hash {
 			log.Infof("FORK: Block %v (height %v) forks the chain at height "+
 				"%d/block %v, but does not cause a reorganize",
-				node.hash,
-				node.height,
-				fork.height,
-				fork.hash)
+				node.hash, node.height, fork.height, fork.hash)
 		} else {
 			log.Infof("EXTEND FORK: Block %v (height %v) extends a side chain "+
-				"which forks the chain at height "+
-				"%d/block %v",
-				node.hash,
-				node.height,
-				fork.height,
-				fork.hash)
+				"which forks the chain at height %d/block %v",
+				node.hash, node.height, fork.height, fork.hash)
 		}
 
-		return false, nil
+		branchLen := node.height - fork.height
```

Comment:
instead of referring it to the "branch length" here and in the comments, why not call it the fork length? The main chain could also be considered a branch (of the genesis block) and fork makes it clear it deviates from another chain.